### ISSUE TITLE
Failed to load compute graph when executing label_image example

### ISSUE THREAD

**Issue by tp6vup54** [tensorflow/tensorflow#351]
after cloning tensorflow on my OSX 10.10.4, I tried to run label_image example in tensorflow/examples/, and followed the instructions.
currently seems fine after built even some warnings were occured but no error.
but when I typed 'bazel-bin/tensorflow/examples/label_image/label_image'
it showed:
E tensorflow/examples/label_image/main.cc:260] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/googlenet_graph.pb'

I saw a file named tensorflow_inception_graph.pb in that folder, and I tried to rename it to googlenet_graph.pb and executed again, and it showed:
I tensorflow/core/common_runtime/local_device.cc:40] Local device intra op parallelism threads: 4
I tensorflow/core/common_runtime/direct_session.cc:60] Direct session inter op parallelism threads: 4
Segmentation fault: 11

then I don't know what I am supposed to try, please show me a way to solve it, thanks.

**Comment by vrv** [comment]
Have you seen the updated instructions from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/README.md, and if so, do they still not help?

**Comment by tp6vup54** [comment]
yeah, actually I just cloned it today.
the only step different is wget, I downloaded it from browser and move it to the data folder, then I ran the unzip command below successfully.
$ unzip tensorflow/examples/label_image/data/inception5h.zip -d tensorflow/examples/label_image/data/

and two bazel command were ran, but still go wrong.

**Comment by ebrevdo** [comment]
Can you run it in gdb and paste a backtrace?

**Comment by petewarden** [comment]
After going through the new download process, it looks like the name of the file has changed from the expected one. I'll update the command-line flag default to point to the correct path here:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/label_image/main.cc#L66
When I run the following command passing the correct path with --graph:
`bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb `
I see the same SEGV fault, so it looks like there's something different between the Android graph file which we're now using here, and what the code here is expecting. I'll keep digging.

**Comment by petewarden** [comment]
It turns out there were two problems. I needed to update the default flag to point to the new labels file location, and I hadn't added error handling for the case when the labels file wasn't found, so it just crashed mysteriously when it tried to print the labels. We'll get that updated in our next push, but for now the workaround is to specify the correct labels file location on the command line. Here's one that I've tested:
`bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb --labels=tensorflow/examples/label_image/data/imagenet_comp_graph_label_strings.txt`

Does that help?

**Comment by fivejjs** [comment]
put graph and label files works for me now.

**Comment by tp6vup54** [comment]
@petewarden I run your command and it works well, thanks for helping me guys!

**Comment by vnkmr7620** [comment]
same method i followed but i am getting error. the program is compiled successfully.
![screenshot from 2015-12-10 02 19 25](https://cloud.githubusercontent.com/assets/10511526/11689044/0c07d79c-9eb6-11e5-9b39-fbee372d8fa5.png)
![screenshot from 2015-12-10 02 19 25](https://cloud.githubusercontent.com/assets/10511526/11689046/0d6c6f4e-9eb6-11e5-85bb-cbcdd8b810f5.png)

**Comment by vrv** [comment]
The latest version of the imagenet graphdef changed the names of the input and output nodes.

1) Fetch the latest source from git and it should work: we updated the source to use the new names.

2) Add the following flags to your current build.

 --input_layer=Mul
--output_layer=softmax

**Comment by vnkmr7620** [comment]
Thanks. This solves my problem

**Comment by loretoparisi** [comment]
@vrv I have the same issue on MacOS X when running the java example

```
$ javac -cp libtensorflow-1.0.0-PREVIEW1.jar LabelImage.java
$ java -cp libtensorflow-1.0.0-PREVIEW1.jar:. -Djava.library.path=./jni LabelImage
$ java -cp libtensorflow-1.0.0-PREVIEW1.jar:. -Djava.library.path=./jni LabelImage ./ ./example-400x288.jpg 
Failed to read [./tensorflow_inception_graph.pb]: ./tensorflow_inception_graph.pb
```

I have the native dylib library and the inceptionv5 model in the path:

```
-rw-r--r--  1 loretoparisi  staff      2628 10 Mar 18:14 LabelImage$GraphBuilder.class
-rw-r--r--  1 loretoparisi  staff      6849 10 Mar 18:14 LabelImage.class
-rw-r--r--@ 1 loretoparisi  staff      7343 10 Mar 18:08 LabelImage.java
-rw-r--r--  1 loretoparisi  staff       262 10 Mar 17:10 download.sh
-rw-r--r--@ 1 loretoparisi  staff     35430 10 Mar 18:12 example-400x288.jpg
-rw-r--r--@ 1 loretoparisi  staff  49937555 19 Nov  2015 inception5h.zip
drwxr-xr-x  4 loretoparisi  staff       136 10 Mar 17:10 jni
-rw-r--r--@ 1 loretoparisi  staff     20314 10 Mar 16:58 libtensorflow-1.0.0-PREVIEW1.jar
-rw-r--r--@ 1 loretoparisi  staff     20992 10 Mar 16:58 libtensorflow-src-1.0.0-PREVIEW1.jar
-r-xr-xr-x@ 1 loretoparisi  staff  90199624  1 Gen  1970 libtensorflow_jni.dylib
```

**Comment by Abduoit** [comment]
Hi all and @vrv @petewarden 

I have installed Tensorflow gpu-support successfully, I test it with python example, it works fine, but I have problem when I test it with C++ example, I run 

`bazel build tensorflow/examples/label_image/...` this works fine

`bazel-bin/tensorflow/examples/label_image/label_image` this give me the follwoing error 

`E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'`

I tried all your suggestions, but same error still there, please help me with it ?????

```
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb --labels=tensorflow/examples/label_image/data/imagenet_comp_graph_label_strings.txt
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image--input_layer=Mul
bash: bazel-bin/tensorflow/examples/label_image/label_image--input_layer=Mul: No such file or directory
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --input_layer=Mul
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb --labels=tensorflow/examples/label_image/data/imagenet_comp_graph_label_strings.txt--input_layer=Mul
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb --labels=tensorflow/examples/label_image/data/imagenet_comp_graph_label_strings.txt--input_layer=Mul --output_layer=softmax
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
jesse@jesse-System-Product-Name:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/tensorflow_inception_graph.pb --labels=tensorflow/examples/label_image/data/imagenet_comp_graph_label_strings.txt--input_layer=Mul--output_layer=softmax
E tensorflow/examples/label_image/main.cc:285] Not found: Failed to load compute graph at 'tensorflow/examples/label_image/data/tensorflow_inception_graph.pb'
```

**Comment by cmbowyer13** [comment]
@Abduoit i'm getting the same couldn't load graph command:

@wing-artemis:~/tensorflow$ bazel build tensorflow/examples/label_imageINFO: Analysed target //tensorflow/examples/label_image:label_image (0 packages loaded).
INFO: Found 1 target...
Target //tensorflow/examples/label_image:label_image up-to-date:
  bazel-bin/tensorflow/examples/label_image/label_image
INFO: Elapsed time: 16.417s, Critical Path: 15.79s
INFO: 2 processes: 2 local.
INFO: Build completed successfully, 3 total actions
@wing-artemis:~/tensorflow$ bazel-bin/tensorflow/examples/label_image/label_image --graph=tensorflow/examples/label_image/data/frozen_inference_graph.pb --labels=tensorflow/examples/label_image/data/object_detection.txt
Fine to this point in main.
2018-07-05 15:39:24.407219: E tensorflow/examples/label_image/main.cc:221] graph_path:~/tensorflow/tensorflow/examples/label_image/data/frozen_inference_graph.pb
2018-07-05 15:39:24.407630: E tensorflow/examples/label_image/main.cc:224] LoadGraph ERROR!!!!Not found: Failed to load compute graph at '~/tensorflow/tensorflow/examples/label_image/data/frozen_inference_graph.pb'

**Comment by cmbowyer13** [comment]
Not sure why it's striking through that comment above either.
